modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -526,7 +526,11 @@ class ApiClient(object):
         """
         if not auth_settings:
             return
-
+        if self.configuration.security_schemes:
+            auth_settings = [
+                value for value in auth_settings
+                if value in self.configuration.security_schemes
+            ]
```

Comment:
Is this still needed if we only return valid fully filled out auth_setting items in from 
self.configuration.auth_settings()?

Reply:
I am removing.